ci/Dockerfile-envoy-distroless

```diff
@@ -0,0 +1,10 @@
+FROM gcr.io/distroless/base-debian10:nonroot
+
+ADD configs/envoyproxy_io_proxy.yaml /etc/envoy/envoy.yaml
+
+ARG ENVOY_BINARY_SUFFIX=_stripped
+ADD linux/amd64/build_release${ENVOY_BINARY_SUFFIX}/* /usr/local/bin/
```

Comment:
given the lack of `sh` etc im  wondering if `$PATH` is set 

might be necessary to hardcode the path to the binary

Reply:
Path is set. Checking layout of image through ":debug" image. The problem is libc version. I managed to build it locally by building via bazel envoy-static + packaging it via Dockerfile-envoy-distroless and it fails on:

```
oleksiyp@pop-os:~/dev/envoy$ docker run a7d14c019784
envoy: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by envoy)
envoy: /lib/x86_64-linux-gnu/libpthread.so.0: version `GLIBC_2.30' not found (required by envoy)
envoy: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by envoy)
```
But maybe it captures my libc from system.

Reply:
Image `distroless-debian9` contains:
```
libc-2.24.so
libm-2.24.so
libpthread-2.24.so
```
while `debian10`:
```
libc-2.28.so
libm-2.28.so
libpthread-2.28.so
```